src/Containers.jsx

```diff
@@ -132,7 +134,7 @@ class Containers extends React.Component {
         }, {
             name: _("Console"),
             renderer: ContainerTerminal,
-            data: { containerId: container.Id, containerStatus: container.State, width:this.state.width, system:container.isSystem }
+            data: { containerId: container.Id, containerStatus: container.State, width:this.state.width, system:container.isSystem, tty: tty }
```

Comment:
not big of a deal - but this is a new property here, maybe you want to add it in PropTypes. (which is missing  half of the others already so we can fix this in the future)

Reply:
ah right, I added the new one and also added the forgotten ones. Thanks for spotting!